### ISSUE TITLE
Feature request: make the tab search automatically select the first item

### ISSUE THREAD

**Issue by Igetin** [Bill13579/tabby#198]
# How it works now

**Step 1:** I press Ctrl+Shift+Space to activate the tab search. The popup looks like this:

<img src="https://github.com/Bill13579/tabby/assets/14016863/73530270-01e6-436d-806d-5683ac05cfda" width="250"/>

**Step 2:** I type “hack”. The popup looks like this:

<img src="https://github.com/Bill13579/tabby/assets/14016863/d27f65cb-7084-4854-9955-04004bbe510d" width="250"/>

**Step 3:** I want to go to the tab that is the first search result. I press enter to “select” the result, and enter again to go to the tab.

# How the feature should work

I would like step 3 to **only require one enter press** to go to the tab. The first item in the list should always be **automatically activated** before I even type anything. If this was the case, pressing enter would automatically go to the first tab, and pressing down would highlight the second result (not the first).

Basically, I’m asking for this to work the same way Chrome’s tab search functionality works. When the tab search shortcut is pressed, the popup shows up and the first item is **already highlighted**:

<img src="https://github.com/Bill13579/tabby/assets/14016863/7e30d71d-8934-48ba-b460-5297658f179a" width="250"/>

When I type “hack”, the Hacker News result is highlighted automatically:

<img src="https://github.com/Bill13579/tabby/assets/14016863/6387011b-65ea-46a6-b5d2-50f814fe3daa" width="250"/>

At this point, I only need to press enter **once** to go to that tab.
